scripts/prepare_csv.py

```diff
@@ -0,0 +1,182 @@
+import json
+import logging
+import sys
+import warnings
+from pathlib import Path
+from typing import Optional
+
+import pandas as pd
+from tqdm import tqdm
+
+warnings.filterwarnings("ignore")
+
+import torch
+from torch.utils.data import random_split
+
+# support running without installing as a package
+wd = Path(__file__).parent.parent.resolve()
+logger = logging.getLogger(__name__)
+sys.path.append(str(wd))
+
+from lit_gpt.tokenizer import Tokenizer
+
+SEED = 42
+MASK_INPUTS = False
+IGNORE_INDEX = -1
+TEST_SPLIT_FRACTION = 0.1
+CHECKPOINT_DIR = Path("checkpoints/stabilityai/stablelm-base-alpha-3b")
+CSV_PATH = "instruction.csv"
+DESTINATION_DIR = Path("data/custom")
+
+
+def prepare(
+    csv_path: Path = CSV_PATH,
+    destination_path: Path = DESTINATION_DIR,
+    checkpoint_dir: Path = CHECKPOINT_DIR,
+    datafile_name: Optional[str] = None,
+    test_split_fraction: float = TEST_SPLIT_FRACTION,
+    seed: int = SEED,
+    mask_inputs: bool = MASK_INPUTS,
+    ignore_index: int = IGNORE_INDEX,
+) -> None:
+    """Prepare dataset for instruction tuning from csv
+
+    Args:
+        llm: str helps to generate dataset with llm specific system prompts. example
```

Comment:
there is no such args as `llm`

Reply:
I should have deleted that comment, this was something I was planing to internally initially but not sure about.